src/main/java/com/spartronics4915/frc2019/Constants.java

```diff
@@ -194,8 +194,8 @@ public static Pose2d getRobotLengthCorrectedPose(Pose2d oldpose)
     public static final int kRampSensorId = 0;
     public static final double kRampSpeed = 1.0;
     public static final double kShootSpeed = 1.0;
-    public static final double kShootTime = 4.0;
-    public static final double kTransitionTime = 1.0;
+    public static final double kShootTime = 2.0;
```

Comment:
Maybe change to `kRampMovingTime`? Also add a comment that makes it clear that these are in seconds, and that `kShootTime` must be greater than `kShootBayDelay` to make the code work.